siem_integrations/README.md

```diff
@@ -26,10 +26,103 @@ In order to utilize splunk2kafka, a [running Kafka instance](https://kafka.apach
 2. Install export2kafka ([Instructions](https://github.com/rapidsai/clx-siem-integration/blob/master/splunk2kafka/export2kafka/README.md))
 
 
-## CLX Query
-
-Coming soon...
-
+## CLX Query Sample Dataset
+
+1. Download sample dataset.
+    
+    ```aidl
+    wget http://files.grouplens.org/datasets/movielens/ml-25m.zip
+    ```
+   
+## CLX_Query_Service
+
+CLX Query Service is a restful service that runs using the django platform, which takes requests from the splunk clx query app and executes blazingsql query and returns the json response.
+
+### Install and Run CLX_Query_Service
+
+1. Install dependencies if not available.
+
+    ```aidl
+    pip install django gunicorn djangorestframework supervisor
+    ``` 
+2. Update settings by adding current query service running hostname.
+
+    ```aidl
+    vi clx_query_service/settings.py
+    ``` 
+    ```aidl
+    ALLOWED_HOSTS = ["<hostname goes here...>"]
```

Comment:
What hostname should be put here? Some more details might be useful.

Reply:
Added detailed information with example.